packages/flutter_hooks/test/use_material_states_controller_test.dart

```diff
@@ -21,10 +21,17 @@ void main() {
       element
           .toDiagnosticsNode(style: DiagnosticsTreeStyle.offstage)
           .toStringDeep(),
-      equalsIgnoringHashCodes(
-        'HookBuilder\n'
-        ' │ useMaterialStatesController: MaterialStatesController#00000({})\n'
-        ' └SizedBox(renderObject: RenderConstrainedBox#00000)\n',
+      anyOf(
+        equalsIgnoringHashCodes(
+          'HookBuilder\n'
+          ' │ useMaterialStatesController: MaterialStatesController#00000({})\n'
+          ' └SizedBox(renderObject: RenderConstrainedBox#00000)\n',
+        ),
+        equalsIgnoringHashCodes(
+          'HookBuilder\n'
+          ' │ useMaterialStatesController: WidgetStatesController#00000({})\n'
```

Comment:
This feels a bit weird to me. `useMaterialStatesController` is explicitly instantiating `MaterialStatesController`

How did we end-up with a `WidgetStatesController`? Wouldn't we get a type error?

Reply:
Oh `MaterialStatesController` is deprecated and it's a typedef

Reply:
It is weird. `MaterialStatesController` is being moved down to the widgets layer and renamed to `WidgetStatesController`.  For migration purposes, there is still a `MaterialStatesController`, but it is just a typedef of `WidgetStatesController`, so it acts the same as how it did before. Except for the debug functions, which return strings mentioning `WidgetState`.